### ISSUE TITLE
Carousel image href does not add quotes to html

### ISSUE THREAD

**Issue by blaaat** [mjmlio/mjml#2242]
**Describe the bug**

double quotes around the href attribute on the <a of an carousel-image are missing. URL's with & fail to compile.

**To Reproduce**
```
<mjml>
  <mj-body >

     <mj-section >
       <mj-column>
                 <mj-carousel thumbnails="hidden">
          <mj-carousel-image href="https://www.google.com?a=1&b=1" src="https://mjml.io/documentation/images/logo.png" />
        </mj-carousel>
       </mj-column>
    </mj-section>
    </mj-wrapper>


  </mj-body>
</mjml>
```

**Expected behavior**

URL should 

**MJML environment (please complete the following information):**
 - OS: [e.g. MacOS]: MacOS
 - MJML Version 4.9.1
 - MJML tool used [e.g MJML App]: cli

**Email sending environment(for rendering issues)**:
 - Platform used to send the email [e.g [Putsmail](https://putsmail.com/)]  

**Affected email clients (for rendering issues):**
 - Email Client [e.g Gmail] 
 - OS: [e.g. Windows]
 - Browser [e.g. Google Chrome]

**Screenshots**
<img width="670" alt="Schermafbeelding 2021-04-25 om 14 12 32" src="https://user-images.githubusercontent.com/277820/115992893-4a1b0f00-a5d0-11eb-8d87-e07ad1acb54e.png">

**Additional context**

**Comment by GarryFlemings** [comment]
@blaaat I'll keep digging for other information. Perhaps you can provide more, too.

https://mjml.io/try-it-live/2C-xozB041a is an attempt to recreate your code. You didn't show a full script, so I filled in some blanks.

The condition you cite is present in the HTML.

When I click on the icon, I get the expected behavior; I land at Google. Maybe just lucky.

New info, please: What effect do you expect? What are you finding?

**Comment by blaaat** [comment]
Sorry, In your example; when you check the "Minify HTML" button, an "MJML badly formatted" error occurs. 

I would expect the output HTML to include the url between quotes. A working workaround is adding 
`href='"http://www.gogole.com/?a=b&c=1"'` like this: https://mjml.io/try-it-live/-H4idc3bnK3

**Comment by iRyusa** [comment]
Seems like this bug was here for a while... nice catch https://github.com/mjmlio/mjml/blob/master/packages/mjml-carousel/src/CarouselImage.js#L143

**Comment by kmcb777** [comment]
Hi, this has just been fixed in 4.9.2
